.cirrus.yml

```diff
@@ -214,6 +214,7 @@ dist_builder_task:
   container:
     dockerfile: contrib/build.Dockerfile
   build_script:
+    - pyenv install 3.9.7
```

Comment:
`contrib/build.Dockerfile` should be changed instead of cirrus.

Reply:
Should this line be dropped entirely then?

Reply:
Yes

Reply:
Done